Add release meta issue template for 1.1.1 (#1100)
opensearch-project/opensearch-build: .github/meta_issue_template/release-1.1.1-plugin-issues.md

Coming from [opensearch-build#870](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
### You can find all the date in above issue 
### (We only make changes to OpenSearch-Dashboards to 1.1.1, therefore, OpenSearch stays 1.1.0.)

> 

Reviewer: Maybe include a 'How to use this issue' section, I think you could steal from https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md?plain=1#L11

### Preparation

- [ ] Assign this issue to a release owner.
- [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.

### CI/CD

> - [ ] Increment version on main to `1.1.1.0`.

Reviewer 2: The version increment would be on 1.1.

Author: Fixed.

- [ ] Ensure working and passing CI.

> - [ ] Re(add) this repo to the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1).

Reviewer 2: Since we're only making a change to Dashboards, reference the actual dashboards manifest, which I think needs to be created. You can do it as part of this PR, mimic how opensearch-1.1.1.yml looks like. Since this is a patch and the newly created .yml will contain distribution references, this line should say "Change the build from distribution to build from source in the 1.1.1 manifest."

Author: I copied from 1.2.0 dashboards manifest so no need for them to change from dist to source, since there is no dist for 1.1.0 dashboards.

### Pre-Release

> - [ ] Branch and build from a `1.1` branch.

Reviewer 2: This should say 1.1.1.

Author: So 1.1.1 branch has 3 digit because it is patch version?
I thought we always use 2 digits to contain both minor and patch releases.
Thanks.

Reviewer 2: I think you're right, we never create a 1.1.1 branch, just fix it on 1.1, since there's no backporting to 1.1.1.

Author: Fixed.

> - [ ] Update your branch in the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1).

Reviewer 2: Link actual dashboards manifest.

Author: Linked.

- [ ] Feature complete, pencils down.

> - [ ] Apply sanity testing, and update results in the comment, contact corresponding assigner in meta issue above if needed.

Reviewer: How about `Integration tests completed`, I do not like the term sanity testing, we should be making sure everything works not just kicking the tires.

The follow up comment that you added, should we put that as a note on top of the ticket instead of as part of this line item?

Author: Make changes, and will keep the current comment location as it is. Since they are logically apply in order.

> - [ ] Fix bugs that target this release.

Reviewer: What do you think about adding `All intermittent test failures have issues filed`?

Author: Add the line.

This is a component issue for release 1.1.1.
Coming from [release issue 1.1.1](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.

<details><summary>How to use this component issue</summary>
<p>

## This Component Issue
This component issue captures the state of the OpenSearch release, on component/plugin level, its assignee is responsible for driving the release of the component.  Please contact them or @mention them on this issue for help. 

## Release Steps

> There are several steps to the release process, these steps are completed as the whole release and components that are behind present risk to the release.  The release owner completes the tasks in this ticket, whereas component owners resolve tasks on their ticket in their repositories.

Author: I didnt change all the languages, just a few. Will try your example here thanks.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

You can find all the corresponding dates of each step in the release issue above.

> ### Component List

Reviewer: Remove this section

Author: Fixed.

To aid in understanding the state of the release there is a table with status indicating each component state in the release issue.  This is updated based on the status of the component issues.

</p>
</details>

### You can find all the date in above issue 
### (We only make changes to OpenSearch-Dashboards to 1.1.1, therefore, OpenSearch stays 1.1.0.)

### Preparation

- [ ] Assign this issue to a release owner.

> - [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).

Reviewer: Just occurred to me while reading this, what would you think about adding?
` - [ ] All the tasks in this issue have been reviewed by the release owner.`

Author: Great, added.

- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.

### CI/CD

> - [ ] Increment plugin version on 1.1 branch to `1.1.1.0` (Only if you have any code changes, else, stay on 1.1.0.0 for your plugin).

Reviewer: ```suggestion
- [ ] If including changes in this release, increment plugin version on 1.1 branch to `1.1.1.0`, otherwise stay on `1.1.0.0`.
```

Author: Fixed.

> - [ ] Make necessary code change if you have any patch for 1.1.1.

Reviewer: `necessary` is subjective, remove the condition since the above statement cover

```suggestion
- [ ] All code change are for 1.1.1 are complete.
```

Author: Fixed.

- [ ] Ensure working and passing CI.
- [ ] Re(add) this repo to the (if not exist) [opensearch-dashboards input manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1/opensearch-dashboards-1.1.1.yml).

### Pre-Release

- [ ] Update your branch in the [opensearch-dashboards input manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1/opensearch-dashboards-1.1.1.yml).
- [ ] Merge any changes to 1.1 you need for the patch
- [ ] Complete integration tests, and update results in the comment, contact corresponding assigner in meta issue above if needed.

> - [ ] Fix bugs that you find during the integration test, wait for a new build and test again.

Reviewer: `wait for a new build and test again.` 😍

Author: Thanks.
